tests/cephfs/ceph-83574161.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import random
+import string
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+
+    """
```

Comment:
Could we also add pre-requisites here

Reply:
@yogesh-mane I have added the pre-requisites